pandas-stubs/core/frame.pyi

```diff
@@ -2279,3 +2279,6 @@ class DataFrame(NDFrame, OpsMixin):
     ) -> Self: ...
     def __truediv__(self, other: float | DataFrame | Series | Sequence) -> Self: ...
     def __rtruediv__(self, other: float | DataFrame | Series | Sequence) -> Self: ...
+
+class _PandasNamedTuple(tuple[Any, ...]):
+    def __getattr__(self, field: str) -> Any: ...
```

Comment:
I'm thinking we should return `Scalar` here, because we also return `Scalar` when someone does `df.loc[3, "a"]`

While it's true that some non-scalar value could be an individual element of a `DataFrame`, I've taken the philosophy of limiting the types to what is "normal" usage, and if you put a funky type in a `DataFrame` or `Series`, then you can do a `cast` to fix it.   I've done that in some of our application code when we have lists or other objects inside a `Series` or `DataFrame`.